service/history/execution/workflow_test.go

```diff
@@ -20,339 +20,320 @@
 
```

Comment:
^ kinda surprised the copyright check didn't complain about this, but you probably want to un-double-comment this